### PR TITLE
MINOR: change placeholder for add asset in teams

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes <issue-number>

- change the placeholder for adding asset in teams for more clarity
- fix the documentation link
- fix the data squashing in the single line, in entity details page.

Issue :   
image
image

image

Resolved :  

image
image

image

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json [hunk 1]
```diff
@@ -1244,6 +1244,7 @@
...
     "add-query-helper-message": "Fügen Sie eine SQL-Abfrage zur Ausführung in der Datenbank hinzu. Die gleiche Abfrage kann mehreren Tabellen hinzugefügt werden, indem Sie aus den Tabellen in der Option 'Verwendete Abfrage' auswählen. Wählen Sie aus, Ihre Abfrage für zukünftige Referenz zu beschreiben.",
     "add-role-message": "Rollen werden Benutzern zugewiesen. In OpenMetadata sind Rollen Sammlungen von Richtlinien. Jede Rolle muss mindestens eine Richtlinie angehängt haben. Eine Rolle unterstützt mehrere Richtlinien mit einer Eins-zu-vielen-Beziehung. Stellen Sie sicher, dass die erforderlichen Richtlinien erstellt sind, bevor Sie eine neue Rolle erstellen. Erstellen Sie umfangreiche Zugriffssteuerungsrollen mit gut definierten Richtlinien basierend auf bedingten Regeln.",
+    "adding-new-asset-to-team": "Clicking on Add Asset will redirect to the Explore page, from where you can assign a Team as the Owner of the Entity.",
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json:1247]:
```suggestion
    "adding-new-asset-to-team": "Click on Add Asset, which will take you to the Explore page. From there, you'll be able to assign a Team as the Owner of the asset.",
```

File: openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx [hunk 2]
```diff
@@ -413,7 +413,11 @@ const AssetsTabs = forwardRef(
             type={ERROR_PLACEHOLDER_TYPE.FILTER}
           />
         );
-      } else if (noDataPlaceholder || searchValue || !permissions.Create) {
+      } else if (
+        isObject(noDataPlaceholder) ||
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx:417]: Hello @Ashish8689 , can you explain why did we need this change?

**Review by Sachin-chaurasiya — COMMENTED** [review]
Left a few comments, thanks

File: openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx:417]: the noDataPlaceholder can be an object or string. and in this particular condition, we are using value only from object. So to further use the noDataPlaceholder as a string have used this condition.


image

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  changes made as per comments and also addressed the above comments!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**